internal/dynamicanalysis/analysis.go

```diff
@@ -45,11 +67,58 @@ func Run(sb sandbox.Sandbox, args []string) (*Result, error) {
 	// Run the command
 	log.Debug("Running dynamic analysis command",
 		"args", args)
+
+	log.Debug("Reroute all http traffic through sslsplit")
+	iptables := exec.Command("iptables", "-t", "nat", "-A", "PREROUTING", "-i", "cni-analysis", "-p", "tcp", "--dport", "80", "-j", "REDIRECT", "--to-port", "8081")
+	err := iptables.Start()
+
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	err = iptables.Wait()
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	log.Debug("Reroute all https traffic through sslsplit")
+	iptables = exec.Command("iptables", "-t", "nat", "-A", "PREROUTING", "-i", "cni-analysis", "-p", "tcp", "--dport", "443", "-j", "REDIRECT", "--to-port", "8080")
```

Comment:
So I think that either these two rules can be added to a `*nat` section in `iptables.rules`.

However, it may be desirable to only optionally enable SSLSplit, in which case it would be ideal if these two rules were removed after the sandbox exits (even better if it had a `-s` option for the containers IP address)